tm2/pkg/db/_all/all.go

```diff
@@ -0,0 +1,13 @@
+// Package all imports all available databases. It is useful mostly in tests.
+//
+// cgo databases (rocksdb, cleveldb) will be excluded if CGO_ENABLED=0.
+package all
+
+import (
+	// Keep in sync with list of non-cgo backends.
+	// Add cgo backends in all_cgo.go.
+	_ "github.com/gnolang/gno/tm2/pkg/db/boltdb"
+	_ "github.com/gnolang/gno/tm2/pkg/db/fsdb"
```

Comment:
Unpopular opinion: we should probably just drop these DBs entirely from the codebase, they are never gonna be used in place of LevelDB in the near future

cc @ajnavarro

Reply:
TBH, because tendermint2 is supposed to be more generic and support different databases for different usecases, I think it makes sense to have support for boltdb, potentially also badger or pebbledb. `fsdb` I'm not so sure about `fsdb`, whose primary documentation is "It's slow." and seems more like an example implementation of the DB interface rather than a real life practical package.